### ISSUE TITLE
travisCI does not build for pull requests

### ISSUE THREAD

**Issue by saketkc** [Homebrew/homebrew-science#1716]
I guess after merging https://github.com/Homebrew/homebrew-science/pull/1679, the travis build is green  for all requests. Probably because of https://github.com/Homebrew/homebrew-science/blob/master/travisCI/travis.sh#L29

```
export changed_files=`git diff-tree --no-commit-id --name-only -r $TRAVIS_COMMIT`
```
I am not really sure, why this does not work out. It could be related to https://github.com/travis-ci/travis-ci/issues/1719 and https://github.com/travis-ci/travis-ci/issues/2666 .

**Comment by dpo** [comment]
Yes I've noticed. One possibility would be to use `set -evx` at the top of `travis.sh` so that all commands and what they evaluate to are printed on the console.

**Comment by saketkc** [comment]
I am working on a solution. Will update.

**Comment by saketkc** [comment]
Do you suggest changing to the `brew test-bot` method? It is I think okay to assume the travis files would not be changed that often.

The reason I say this is because `test-bot` handles what I am trying to use, it fetches the difference between the origin and upstream branch.

**Comment by dpo** [comment]
Were you able to test it? I wish the test-bot build could be verbose. How do we get the log in case of failure?

**Comment by saketkc** [comment]
Closed by #1717

**Comment by saketkc** [comment]
This was due to https://github.com/blog/1941-organization-approved-applications

Mike has re-enabled it.
